code/game/objects/items.dm

```diff
@@ -879,7 +879,9 @@
 
 ///Returns the temperature of src. If you want to know if an item is hot use this proc.
 /obj/item/proc/get_temperature()
-	return heat
+	. = heat
+	if(resistance_flags & ON_FIRE)
+		. = max(., FIRE_MINIMUM_TEMPERATURE_TO_EXIST)
```

Comment:
```suggestion
	if(resistance_flags & ON_FIRE)
		return max(heat, FIRE_MINIMUM_TEMPERATURE_TO_EXIST)
	return heat
```

No trivial use of `.` -- https://github.com/tgstation/tgstation/blob/master/.github/guides/STANDARDS.md#dot-variable-